packages/ui-components/src/components/text-area/text-area.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/**
+	* @prop --height-default: The height of the text area when there's no text
+	* @prop --height-active: The height of the text area when there's text
+	*/
+	--height-default: 21px;
+	--height-active: 84px;
+}
+
+.text-area-container {
+	display: flex;
+	gap: $spacing-2x;
+
+	.left-icon {
+		margin-top: $spacing-2x;
+	}
+
+	.text-area {
+
```

Comment:
`-`

Reply:
I think there's a problem with the linter, this is the result after `lint:scss:fix` in this case, I'll try to see if it clears the linter step when removing the space.